### PR TITLE
Included twitter bootstrap, applied bootstrap layout on layouts/appli…

### PR DESCRIPTION
**Pull request by osvaldoFernandez92**
Summary
Twitter bootstrap was applied. Card developed: https://trello.com/c/wdwKa1N4 . 
Layout generated in html.erb and then modified into html.slim.
Bootstrap_and_overrides.css.less included in assets/stylesheets

### CODE REVIEW

**Comment by osvaldoFernandez92** [comment]
@epintos

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -1,10 +1,52 @@
...
+          ul.nav.navbar-nav
+    
+    .container
+      .row
+        .col-lg-9
+          = bootstrap_flash
+          = yield
+        .col-lg-3
+
+        /! /span
```
Comment (epintos) [inline app/views/layouts/application.html.slim]: Why you didnt put `span` directly?

**Comment by epintos** [comment]
@osvaldoFernandez92

**Comment by epintos** [comment]
Squash and :+1:

**Pull request merged**